core/test/martian/yaml_test.clj

```diff
@@ -85,7 +80,8 @@
            (-> openapi-yaml
                (yaml/cleanup)
                deprecate-all
-               (openapi->handlers {})
+               (openapi->handlers {:encodes ["application/xml"]
+                                   :decodes ["application/json"]})
```

Comment:
This doesn't affect the thing under test, but gets rid of some false-positive warnings in the test logs.